Space.js

```diff
@@ -20,15 +20,36 @@ module.exports = function Space(key, reporters, errback) {
   };
 
   this.meter = func => {
-    if (typeof func !== 'function') {
+    if (typeof func !== 'function' && !isPromise(func)) {
       throw new Error('must pass a function as argument');
     }
 
+    if (isPromise(func)) {
```

Comment:
I think you can do the same thing when `isPromise or isAsync`.

Reply:
How?
The internet perter identifies them differently (there are tests for that)